cli/src/semgrep/commands/ci.py

```diff
@@ -418,6 +408,18 @@ def ci(
         else:
             run_install_semgrep_pro()
 
+        output_settings = OutputSettings(
+            output_format=output_format,
+            output_destination=output,
+            verbose_errors=verbose,
+            timeout_threshold=timeout_threshold,
+            output_time=time_flag,
+            output_per_finding_max_lines_limit=max_lines_per_finding,
+            output_per_line_max_chars_limit=max_chars_per_line,
+            dataflow_traces=dataflow_traces,
+        )
```

Comment:
Should this be in the `if engine_type.is_pro` block? Seems like this would not be defined below as-is.

Reply:
Well there is a slight mistake. Good catch.